scripts/l2/l2-teardown.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -euo pipefail
+
+# Path to the .env file
+ENV_FILE="$(pwd)/.env"
+
+# Source the .env file
+source "$ENV_FILE"
+
+# Function to send out funds from an address
+drain_address() {
+    local address=$1
+    local private_key=$2
+
+    # Get existing balance
+    balance=$(cast balance --rpc-url "$L1_RPC_URL" "$address")
+    echo "Existing balance: $balance"
+
+    if [[ "$balance" == "0" ]]; then
+        echo "Balance is 0, skipping..."
+        return
+    fi
```

Comment:
this check seems unnecessary b/c

- we have the amount > 0 check below
- the balance will never be 0. if it's 0, it means sth is wrong